R/FilteredData.R

```diff
@@ -60,6 +62,7 @@
 #'
 #' @examplesIf requireNamespace("MultiAssayExperiment")
 #' ### set_filter_state
+#' library(shiny)
 #'
 #' utils::data(miniACC, package = "MultiAssayExperiment")
```

Comment:
Do we need a `utils::` in an example?

Reply:
Missed that one as I removed the others. Good catch.